spec/support/auth_helper.rb

```diff
@@ -9,6 +9,32 @@ def login_as(user)
     session[:group]   = user.current_group_id
     user
   end
+
+  # TODO: Refactor, stub specific features, document
+  def stub_user(features:)
+    user = FactoryGirl.build(:user_with_group)
+    allow(controller).to receive(:current_user).and_return(user)
+    allow(User).to receive(:current_user).and_return(user)
+    allow(User).to receive(:server_timezone).and_return("UTC")
+    allow_any_instance_of(described_class).to receive(:set_user_time_zone)
+
+    stub_bool = case features
+                when :all
+                  true
+                when :none
+                  false
+                else
+                  raise ArgumentError, "Unknown features option." +
```

Comment:
Prefer `\` line continuation over `+` to avoid creating 2 strings (compile time concatenation vs runtime)